Data-Engineering/Airflow/exmample_spark_pi_oss.py

```diff
@@ -0,0 +1,60 @@
+from airflow import DAG
+from airflow.models.param import Param
+from airflow.providers.cncf.kubernetes.operators.spark_kubernetes import (
+    SparkKubernetesOperator,
+)
+from airflow.providers.cncf.kubernetes.sensors.spark_kubernetes import (
+    SparkKubernetesSensor,
+)
+from airflow.utils.dates import days_ago
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": days_ago(1),
+    "email": ["airflow@example.com"],
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "max_active_runs": 1,
+    "retries": 0,
+}
+
+dag = DAG(
+    "spark_pi_oss",
+    default_args=default_args,
+    schedule_interval=None,
+    tags=["ezaf", "spark", "pi"],
+    params={
+        "Spark-image-url": Param(
```

Comment:
nit: can we use a snake case or camel case for name of the param? Like `spark_image_url`, to be consistent with other dags